src/requirements/Github/components/GithubCommitCount.tsx

```diff
@@ -0,0 +1,97 @@
+import {
+  FormControl,
+  FormErrorMessage,
+  FormLabel,
+  NumberDecrementStepper,
+  NumberIncrementStepper,
+  NumberInput,
+  NumberInputField,
+  NumberInputStepper,
+} from "@chakra-ui/react"
+import { ControlledTimestampInput } from "components/common/TimestampInput"
+import { useController, useFormState } from "react-hook-form"
+import { RequirementFormProps } from "requirements"
+import parseFromObject from "utils/parseFromObject"
+
+const GithubCommitCount = ({ baseFieldPath }: RequirementFormProps) => {
+  const { errors } = useFormState()
+
+  const {
+    field: { ref, value, onChange, onBlur },
+  } = useController({
+    name: `${baseFieldPath}.data.id`,
+    rules: {
+      required: "This field is required.",
+      min: {
+        value: 1,
+        message: "Amount must be greater than 0",
+      },
+    },
+  })
+
+  return (
+    <>
+      <FormControl
```

Comment:
These lines (34-64) are exactly the same as the content of the `GithubAccountAge` component - so you could just reuse that component here in my opinion. DRY ;)